src/DataStructures/Python/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ set(LIBRARY "PyDataStructures")
 
 spectre_python_add_module(
   DataStructures
-  SOURCES Bindings.cpp DataVector.cpp
+  SOURCES Bindings.cpp DataVector.cpp Matrix.cpp
```

Comment:
Could you change this to now be:
```cmake
  SOURCES
  Bindings.cpp
  DataVector.cpp
  Matrix.cpp
```
? The reason is this way if two people add or modify the list, it's less likely to result in a conflict :)